src/CAN/FakeCANBoard.cpp

```diff
@@ -160,11 +160,14 @@ int main() {
 			robot::types::datatime_t currTime = robot::types::dataclock::now();
 
 			// get y data: set point (target vel * time since set velocity call) + initial pos
-			double setPoint = (targetVel * util::durationToSec(currTime - startTime)) +
-							  initialMotorPos.getData();
+			[[maybe_unused]] double setPoint =
```

Comment:
No point in marking these variables as unused if we immediately print it, so we should remove those attributes for `setPoint` and `motorPos`